docs/custom-matchers/writing-custom-matcher.md

```diff
@@ -15,8 +16,9 @@ const endsWithMatcher = pattern => {
       const wrapRegExp = new RegExp(`(.*)\.${pattern}$`); // In production application pattern should definitely be escaped!
 
       return {
-        unwrap: pattern,
-        wrap: action.type.match(wrapRegExp)[1],
+        id: `ends-with${pattern}`,
```

Comment:
Technically it does not matter, because this ID just needs to be unique, so unless somebody writes custom matcher which generates the same ID everything's fine.